app/models/media.rb

```diff
@@ -22,6 +21,10 @@ def class_name
     'Media'
   end
 
+  def projects
+    ProjectMediaProject.where(project_media_id: self.project_medias.map(&:id)).map(&:project)
+  end
+
   def get_team
```

Comment:
Don't implement via `projects` but via `project_medias.each team`

Reply:
and rename to `:team`

Reply:
Got the teams via project_medias relation and renamed it to `get_teams` as media can exist in multiple teams